share/spice/congress/footer.handlebars

```diff
@@ -0,0 +1,4 @@
+{{#if district}}
+    <span class="tile__sub">{{district}} District</span>
```

Comment:
if you use `<div>` with class `tx-clr--grey` you can ditch the `tile__sub` and the extra CSS to adjust the margin